go/kbfs/libkbfs/prefetcher_test.go

```diff
@@ -1372,7 +1373,8 @@ func TestSyncBlockCacheWithPrefetcher(t *testing.T) {
 		Mode: keybase1.FolderSyncMode_ENABLED,
 	})
 	require.NoError(t, err)
-	q.TogglePrefetcher(true, prefetchSyncCh, nil)
+	p2Ch := q.TogglePrefetcher(true, prefetchSyncCh, nil)
+	defer func() { <-p2Ch }()
```

Comment:
@keybase/kbfs-hackers PTAL here

Reply:
Is the idea here to wait until Shutdown happens before finishing the test?

Reply:
Yep

Reply:
Actually I added similar checks to all the tests in `prefetcher_test.go`, since I think they were all susceptible.